Dockerfile.kube-dns

```diff
@@ -14,7 +14,6 @@
 
 FROM ARG_FROM
 
-MAINTAINER Bowei Du <bowei@google.com>
-
 ADD bin/ARG_ARCH/ARG_BIN /ARG_BIN
+USER nobody:nogroup
```

Comment:
oh, didn't think about that, just harmonized the Dockerfiles automatically, will fix